mypy/nodes.py

```diff
@@ -1413,8 +1411,8 @@ def __init__(self, operators: List[str], operands: List[Expression]) -> None:
         self.operands = operands
         self.method_types = []
         self.literal = min(o.literal for o in self.operands)
-        self.literal_hash = ((cast(Any, 'Comparison'),) + tuple(operators) +
-                             tuple(o.literal_hash for o in operands))
+        self.literal_hash = (('Comparison',) + tuple(operators) +
+                             tuple(o.literal_hash for o in operands))  # type: ignore
```

Comment:
This was supposed to be another way to spell the same thing, but it's not since Tuple[Any, ...] is still informative. I should revert this.